experimental/Schemes/Types.jl

```diff
@@ -595,23 +595,10 @@ identifications given by the glueings in the `default_covering`.
     )
     OOX = StructureSheafOfRings(X)
 
+    ### Production of the rings of regular functions; to be cached
     function production_func(F::AbsPreSheaf, U::AbsSpec)
       # If U is an affine chart on which the ideal has already been described, take that.
       haskey(ID, U) && return ID[U]
-      # The ideal sheaf has to be provided on at least one dense
```

Comment:
why is this removed?

Reply:
Because it is taken care of in the respective constructor. This was unacceptably slow and forced computation of glueings for almost no reason.